shortcuts/shortcuts.go

```diff
@@ -55,6 +55,8 @@ var (
 		globalScope, tcell.NewEventKey(tcell.KeyRune, 's', tcell.ModAlt))
 	FocusPrivateChatPage = addShortcut("focus_private_chat_page", "Focus private chat page",
 		globalScope, tcell.NewEventKey(tcell.KeyRune, 'p', tcell.ModAlt))
+	FocusPreviousChannel = addShortcut("focus_previous_channel", "Focus previous channel",
```

Comment:
This shouldn't be named `Focus` but `OpenPreviousChannel` or `SwitchToPreviousChannel`, as focus is used in terms of component focus.